trin-core/src/jsonrpc/handlers.rs

```diff
@@ -22,44 +24,56 @@ pub struct JsonRpcHandler {
 
 impl JsonRpcHandler {
     pub async fn process_jsonrpc_requests(mut self) {
-        while let Some(request) = self.portal_jsonrpc_rx.recv().await {
-            let response: Value = match request.endpoint {
+        while let Some(portal_jsonrpc_req) = self.portal_jsonrpc_rx.recv().await {
+            let response: Value = match portal_jsonrpc_req.endpoint {
                 TrinEndpoint::Discv5Endpoint(endpoint) => match endpoint {
```

Comment:
Assuming that we plan to support more `discv5_*` endpoints, it's probably about time to break out this match into its own function that handles the fetching of data for `Discv5Endpoint`. Maybe adding something like a `Discv5RequestHandler` to maintain consistency with the other subdomains?